src/content/resources/concepts/layer/control/profile/_index.md

```diff
@@ -15,8 +15,8 @@ aliases:
 
 | Profile Schema | Profile Converters | Reference |
 |:--- |:--- |:--- |
-| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/json#converting-oscal-xml-content-to-json)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-index/) |
-| [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_schema.xsd) | [JSON to XML Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_json-to-xml-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-index/) |
+| [JSON Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_schema.json) | [XML to JSON Converter](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_xml-to-json-converter.xsl)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/json-index/) |
+| [XML Schema](https://pages.nist.gov/OSCAL-Reference/release-assets/latest/oscal_profile_schema.xsd) | [JSON to XML Converter](https://github.com/usnistgov/OSCAL/blob/main/build/README.md#converters)<br />([How do I use this?](https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml)) | [Outline](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-outline/)<br />[Reference](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-reference/)<br />[Index](https://pages.nist.gov/OSCAL-Reference/models/latest/profile/xml-index/) |
```

Comment:
the link https://github.com/usnistgov/OSCAL/tree/main/xml#converting-oscal-json-content-to-xml is broken. Also it appears that the hyperlink included for "JSON to XML Converted" should not have been replaced. It points now to the How to page instead of the converter.

Reply:
I am dropping this change and fixing in rebase of commit https://github.com/usnistgov/OSCAL-Pages/pull/60/commits/eb7280b764a78f2291d627384d6711c7048deda4 . I will update the rest of them in bulk.